core/types.go

```diff
@@ -264,10 +264,30 @@ func RenewalInfoSimple(issued time.Time, expires time.Time) *RenewalInfo {
 	validity := expires.Add(time.Second).Sub(issued)
 	renewalOffset := validity / time.Duration(3)
 	idealRenewal := expires.Add(-renewalOffset)
+	windowStart := idealRenewal.Add(-24 * time.Hour)
+	windowEnd := idealRenewal.Add(24 * time.Hour)
+	now := time.Now().In(time.UTC)
```

Comment:
Having this function compute `now`, while the other similar helpers take `now` as an argument, doesn't feel right. They should all behave the same way -- either this one should take `now` as an argument, or the others should have their single argument removed.